src/components/button/stories/common.ts

```diff
@@ -44,6 +44,9 @@ export const DAIKIN_BUTTON_ARG_TYPES = {
   rightIcon: {
     type: "string",
   },
+  onClick: {
+    name: "",
+  },
```

Comment:
It would be good if we had comments.

```suggestion
  // Hide event listeners
  onClick: { name: "" },
```